test/test_io.py

```diff
@@ -171,7 +171,7 @@ def _test_3_load_and_save_is_identity(self):
         self.assertEqual(sample_rate, sample_rate2)
         os.unlink(output_path)
 
-    @unittest.skipIf(set(["sox", "soundfile"]) not in set(BACKENDS), "sox and soundfile are not available")
+    @unittest.skipIf(any(be not in BACKENDS for be in ["sox", "soundfile"]), "sox and soundfile are not available")
```

Comment:
Good catch! `set(["sox", "soundfile"]).issubset(BACKENDS)` is more readable though.

Reply:
I think that's subjective.